macro/o2sim.C

```diff
@@ -108,6 +108,23 @@ FairRunSim* o2sim_init(bool asservice)
   TStopwatch timer;
   timer.Start();
 
+  o2::detectors::DetID::mask_t detMask{};
+  {
+    auto& modulelist = o2::conf::SimConfig::Instance().getActiveDetectors();
```

Comment:
just to be sure I understand correctly: Why do you put this code into braces ? ... To be somewhat isolated?  ... This looks like a function body but is not.

Reply:
just to isolate it, the braces are not really necessary.